modules/mediawiki/files/bin/deploy-mediawiki.py

```diff
@@ -149,11 +149,15 @@ def _construct_rsync_command(time: str, dest: str, recursive: bool = True, local
     raise Exception(f'Error constructing command. Either server was missing or {location} != {dest}')  # noqa: R503
 
 
-def _construct_git_pull(repo: str, submodules: bool = False) -> str:
+def _construct_git_pull(repo: str, submodules: bool = False, branch: Union[bool, str] = False) -> str:
     if submodules:
```

Comment:
Why? I assume you're trying to except False. Use None and type check it to be optional.